### ISSUE TITLE
RFbridge doesnt update alarm sensor when disconnected from Internet (local)

### ISSUE THREAD

**Issue by zd3sf** [AlexxIT/SonoffLAN#1379]
RFbridge 2, firmware 3.5.2. integration (in Local or Auto). with Internet access for the bridge denied from the router, the buttons keep working but the window sensor doesn't update. I found that the bridge bridge connects to this IP (13.56.143.44) even with pihole block, and it stays online in the ewelink app.  Interestingly, if I allow internet access then pull the WAN cord out after the bridge initialized, the sensors keep working. 

I am thinking that the bridge uses the connection to 13.56.143.44 to pull the clock time because the sensor triggers need "last triggered" which needs the clock on the bridge to be correct. What if we spoof this IP address and redirect it to a local NTP server? It wouldn't be this easy because this IP is not only n NTP but is also the connection to ewelink. But if anyone can figure how supply the time clock to sonoff devices without ewelink, that would be awesome... potentially solve all issues with sensors not working in local mode.

![image](https://github.com/AlexxIT/SonoffLAN/assets/46410745/033bd298-7f42-4420-9ab5-7433c4984dba)

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/SonoffLAN?tab=readme-ov-file#issues

**Comment by zd3sf** [comment]
There's nothing in the logs that would be helpful. This a problem between the bridge and the server. This is more a development suggestion sorry

**Comment by AlexxIT** [comment]
Without device and any debug info I can't do anything

**Comment by zd3sf** [comment]
[home-assistant_sonoff_2024-04-13T15-12-22.530Z.log](https://github.com/AlexxIT/SonoffLAN/files/14967776/home-assistant_sonoff_2024-04-13T15-12-22.530Z.log)
[config_entry-sonoff-af35e4956ec234642bc36eb0ae423546.json](https://github.com/AlexxIT/SonoffLAN/files/14967777/config_entry-sonoff-af35e4956ec234642bc36eb0ae423546.json)
Sorry here you go

**Comment by AlexxIT** [comment]
I can see local updates in your logs. You don't have them when there's no internet?

```
2024-04-13 10:12:17.263 DEBUG (MainThread) [custom_components.sonoff.core.ewelink] 100205192c <= Local3 | 192.168.2.175:8081 | {'sledOnline': 'on', 'arming': True, 'rfTrig9': '2024-04-13T14:58:59.000Z', 'ssid': 'TP-LINK_E8A8', 'bssid': '8c:3b:ad:0f:8b:99'} | 682
2024-04-13 10:12:18.083 DEBUG (MainThread) [custom_components.sonoff.core.ewelink] 100205192c <= Local3 | 192.168.2.175:8081 | {'sledOnline': 'on', 'arming': True, 'rfTrig9': '2024-04-13T14:58:59.000Z', 'ssid': 'TP-LINK_E8A8', 'bssid': '8c:3b:ad:0f:8b:99'} | 682
```

**Comment by zd3sf** [comment]
With no internet

`2024-04-13 10:12:16.541 DEBUG (MainThread) [custom_components.sonoff.core.ewelink.cloud] CLOUD None => None
2024-04-13 10:12:16.902 DEBUG (MainThread) [custom_components.sonoff] 1 devices loaded from Cloud
2024-04-13 10:12:16.911 DEBUG (MainThread) [custom_components.sonoff.core.ewelink] 100205192c UIID 0028 | {'bindInfos': {}, 'version': 8, 'only_device': {'ota': 'success'}, 'sledOnline': 'on', 'ssid': 'TP-LINK_E8A8', 'bssid': '8c:3b:ad:0f:8b:99', 'init': 1, 'fwVersion': '3.5.2', 'rssi': -21, 'staMac': '34:98:7A:C3:CE:78', 'setState': 'arm', 'rfList': [{'rfChl': 0, 'rfVal': '57C602F6095E00C301'}, {'rfChl': 1, 'rfVal': '5E7E0308094C00C303'}, {'rfChl': 2, 'rfVal': '424402640738BB809E'}, {'rfChl': 3, 'rfVal': '498802680736BB8097'}, {'rfChl': 4, 'rfVal': '49460264073291A897'}, {'rfChl': 5, 'rfVal': '42080264073291A89E'}, {'rfChl': 6, 'rfVal': '42A2026A07408DE097'}, {'rfChl': 7, 'rfVal': '49E0026807428DE09E'}, {'rfChl': 8, 'rfVal': '49E6026A074052C897'}, {'rfChl': 9, 'rfVal': '49DA026A073E52C89E'}], 'cmd': 'trigger', 'timers': [], 'rfTrig0': '2024-04-12T12:05:04.000Z', 'rfTrig1': '2024-04-10T12:46:06.000Z', 'rfTrig3': '2024-04-12T22:08:29.000Z', 'rfTrig5': '2024-04-12T21:58:56.000Z', 'rfTrig2': '2024-04-13T14:44:49.000Z', 'rfChl': 1, 'rfTrig4': '2024-04-12T21:59:01.000Z', 'rfTrig6': '2024-04-13T13:16:17.000Z', 'rfTrig7': '2024-04-13T13:16:14.000Z', 'rfTrig8': '2024-04-12T20:01:32.000Z', 'rfTrig9': '2024-04-13T14:58:59.000Z'}
2024-04-13 10:12:16.913 DEBUG (MainThread) [custom_components.sonoff] LOCAL mode start
2024-04-13 10:12:17.263 DEBUG (MainThread) [custom_components.sonoff.core.ewelink] 100205192c <= Local3 | 192.168.2.175:8081 | {'sledOnline': 'on', 'arming': True, 'rfTrig9': '2024-04-13T14:58:59.000Z', 'ssid': 'TP-LINK_E8A8', 'bssid': '8c:3b:ad:0f:8b:99'} | 682
2024-04-13 10:12:18.083 DEBUG (MainThread) [custom_components.sonoff.core.ewelink] 100205192c <= Local3 | 192.168.2.175:8081 | {'sledOnline': 'on', 'arming': True, 'rfTrig9': '2024-04-13T14:58:59.000Z', 'ssid': 'TP-LINK_E8A8', 'bssid': '8c:3b:ad:0f:8b:99'} | 682
2024-04-13 10:12:19.915 DEBUG (MainThread) [custom_components.sonoff] Add 3 entities
2024-04-13 10:12:20.623 WARNING (SyncWorker_23) [py.warnings] /usr/local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.1.1'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(`

**Comment by zd3sf** [comment]
With no internet 
[home-assistant_sonoff_2024-04-13T15-54-10.296Z-no internet.log](https://github.com/AlexxIT/SonoffLAN/files/14967936/home-assistant_sonoff_2024-04-13T15-54-10.296Z-no.internet.log)

**Comment by AlexxIT** [comment]
Yes. Same TS will be skipped. I don't know if it safe to remove this check. Then another check would have to be made for local+cloud messages.

https://github.com/AlexxIT/SonoffLAN/blob/08070c312b52d877ffdfb31c73b91d696def4926/custom_components/sonoff/remote.py#L128-L130

**Comment by AlexxIT** [comment]
I don't know how to fix this on the component side. Disabling the date check doesn't seem to be safe. You can also see that the message counter on the device side doesn't increase.

**Comment by zd3sf** [comment]
Pulling the WAN cable out the router while everything else is connected, the RF bridge continue to work normally for 2-3 minutes. Then it misbehaves
